Resources/Prototypes/Catalog/Fills/Lockers/heads.yml

```diff
@@ -33,6 +33,7 @@
   components:
   - type: StorageFill
     contents:
+      - id: ClothingOuterArmorCaptainCarapace
```

Comment:
if they spawn with it now it shouldnt also be in the locker right? this should probably be his hardsuit instead?

Reply:
It's a spare carapace, it's pretty par the course (i.e. [/tg/station](https://github.com/tgstation/tgstation/blob/4152bf9ddac0384d41032c8da6715f6a25abff63/code/game/objects/items/storage/garment.dm#L49) has a spare in their locker). Hardsuits are already in the locker that hasn't been changed, although they should probably be moved to a suit storage unit when/if they get added.